dynamodb/dynamodb-client.w

```diff
@@ -0,0 +1,196 @@
+bring "./dynamodb-types.w" as dynamodb_types;
+
+interface DocumentClient {
+  inflight batchGet(input: Json): Json;
+  inflight batchWrite(input: Json): Json;
+  inflight delete(input: Json): Json;
+  inflight get(input: Json): Json;
+  inflight put(input: Json): Json;
+  inflight query(input: Json): Json;
+  inflight scan(input: Json): Json;
+  inflight transactGet(input: Json): Json;
+  inflight transactWrite(input: Json): Json;
+  inflight update(input: Json): Json;
+}
+
+struct Credentials {
+  accessKeyId: str;
+  secretAccessKey: str;
+}
```

Comment:
Yes, we need to pass fake credentials to the DynamoDB container in the sim, otherwise the client will try to gather the credentials from environment variables. For example, this is what I get:

```
The Wing Console is running at http://localhost:3000/
Error: The SSO session associated with this profile has expired. To refresh this SSO session run aws sso login with the corresponding profile.
   --> dynamodb-client.w:66:12
```